src/oas3/transformers/content.ts

```diff
@@ -108,17 +108,22 @@ export function translateHeaderObject(headerObject: unknown, name: string): Opti
 export function translateMediaTypeObject(mediaObject: unknown, mediaType: string): Optional<IMediaTypeContent> {
   if (!isDictionary(mediaObject)) return;
 
-  const { schema, example, examples, encoding } = mediaObject;
+  const { schema, encoding } = mediaObject;
+
+  const jsonSchema = schema
+    ? (toJsonSchema(schema, {
+        cloneSchema: true,
+        strictMode: false,
+        keepNotSupported: ['example', 'deprecated', 'readOnly', 'writeOnly', 'xml', 'externalDocs'],
+      }) as JSONSchema4)
+    : undefined;
+
+  const example = mediaObject.example || jsonSchema?.example || jsonSchema?.['x-example'];
+  const examples = mediaObject.examples || transformSchemaExamples(jsonSchema?.examples || jsonSchema?.['x-examples']);
```

Comment:
@marbemac @philsturgeon I really think we should NOT support or encourage `x-example` in OpenAPI 3.0. 

While I _kind_ of understand the OAS2 workaround, OAS3 has a clear solution for this and we should not invent our own methods here.

Reply:
I added it as it might be included even if it's not supported, but yeah, perhaps better not to encourage using something that can be done better.

Reply:
Agree - anywhere there is now an official way of accomplishing xyz in openapi 3, we should remove our custom extensions. So if there is `example` in this location, we don't need `x-example`.